cob_frame_tracker/CHANGELOG.rst

```diff
@@ -57,6 +66,61 @@ Changelog for package cob_frame_tracker
 * Contributors: ipa-fxm, ipa-fxm-cm, ipa-fxm-mb
 
+0.6.12 (2016-10-10)
+-------------------
+* 0.6.11
+* update changelog
+* add frameExists check to FrameTracker
+* updateMarker on startLookat
+* cleanup roslint
+* introduce cfg-parameter enable_abortion_checking
+* Added waitForTransform in the getTransform function.. It threw an transform exception
+* Merge branch 'refactor_profile_generator' of github.com:ipa-fxm-cm/cob_control into test_new_cartesian_controller
+  Conflicts:
+  cob_frame_tracker/src/cob_frame_tracker.cpp
+* parameterizable scaling_factor
+* Linear interpolation works fine now. There's still a bug in quaternion interpolation.
+* only reset lookat extension
+* prepare interactive_frame_target for being used with lookat
+* prepare frame_tracker for being used with lookat
+* constant publish rate
+* working on log output
+* parameterizable marker_scale
+* re-activate publishHoldTwist, fix typo
+* publish ZeroTwist, root/tip frame selection
+* introduce scaling_factor and dead_man
+* simple spacenav commander
+* temporarily undo publishHoldTwist
+* 0.6.10
+* update changelog
+* 0.6.9
+* update changelog
+* Corrections integrated from PR: [WIP] Finalizing/Testing of TwistController features (`#51 <https://github.com/ipa320/cob_control/issues/51>`_). Renaming from frame_to_collision to link_to_collision.
+* - Made some changes for test. - Decreased Duration time for markers. - Corrected pose update for self collision check frames. - Added new scripts.
+* boost revision
+* Some preparations for test: IMarker smaller, Alpha settings, More scripts. Default value for Frame Tracker params.
+* - Added functionality to hold twist in case of deviation of cart. distance gets to large.
+* add actionlib
+* explicit dependency to boost
+* more fixes for migration afer merge
+* remove trailing whitespaces
+* migrate to package format 2
+* review dependencies
+* updates from ipa320
+* cleanup
+* sort dependencies
+* review dependencies
+* Fixed bugs in cartesian_controller: waitFor last available transform else extrapolation error; send always a new constructed StampedTransform instead of using an already existent one, else end-effector is decoupled from manipulator and other confusing things happen...; Added responsible node to tf error msg.
+* code styling cob_frame_tracker
+* Implemented proposals from discussion https://github.com/ipa320/cob_control/pull/38. Removed tabs. Corrected node handles.
+* update changelog
+* Removed tracking error publisher / subscriber and removed additional p gain for PD-Control (already done in FrameTracker with PID controller)
+* bug fix
+* - Added a publisher for the tracking errors to send them to cob_twist_controller
+  - Added a subscriber to collect the errors and put them to the solver.
+  - Added a parameter to set the p gain. If 0.0 old behavior is active (default value).
+* Contributors: Marco Bezzon, ipa-fxm, ipa-fxm-cm, ipa-fxm-mb
+
```

Comment:
All this here is wrong, there still is the duplicate 0.6.12 tag, and this one includes 0.6.9 to 0.6.11